test/spmd/test_train_spmd_imagenet.py

```diff
@@ -67,6 +64,7 @@
 from torch_xla.distributed.fsdp.wrap import (recursive_wrap,
                                              transformer_auto_wrap_policy)
 from torch_xla.distributed.fsdp.utils import checkpoint_module
+from torch_xla import runtime as xr
```

Comment:
hmm, why do we need to import if there is no code using it below?

Reply:
We use it for building the device mesh: `    num_devices = xr.global_device_count()`. When migrating `pjrt` to `runtime`, the references were updated but the import was missed